src/Stratis.Bitcoin.IntegrationTests/Mempool/MemoryPoolTests.cs

```diff
@@ -160,20 +160,20 @@ public void TxMempoolBlockDoublespend()
 
                 // Test 1: block with both of those transactions should be rejected.
                 var tipBeforeBlockCreation = stratisNodeSync.FullNode.Chain.Tip;
-                Assert.Throws<ConsensusException>(() => { Block block = TestHelper.GenerateBlockManually(stratisNodeSync, spends); });
+                Assert.Throws<ConsensusException>(() => { Task.Delay(2).Wait(); Block block = TestHelper.GenerateBlockManually(stratisNodeSync, spends); });
```

Comment:
Ah awesome, did adding a delay here fix the test?

Reply:
I think it did yes (it worked in debug mode)